convert.sh

```diff
@@ -95,14 +102,19 @@ get_out_file_diff () {
 }
 
 main () {
-    while getopts ":h" option
+    local enable_build=false
+    while getopts ":hb" option
     do
         case $option in
             h) # help
                 usage
                 return 0
                 ;;
 
+            b) # build
```

Comment:
Perhaps add a `--build` alias?

Reply:
Using `getopts`, I can only parse "short" parameters. I don't really want to make this script more complicated.